content/2-templating/5-dom-ref/angular/inputfocused.component.ts

```diff
@@ -1,14 +1,14 @@
-import { Component, ViewChild, ElementRef, AfterViewInit } from '@angular/core';
+import { Component, ViewChild, ElementRef, OnInit } from '@angular/core';
 
 @Component({
 	selector: 'app-inputfocused',
-	template: '<input type="text" #inputelement />',
+	template: `<input type="text" #inputRef />`,
 })
-export class InputfocusedComponent implements AfterViewInit {
-	@ViewChild('inputelement')
-	inputElement: ElementRef<HTMLInputElement> | undefined;
+export class InputfocusedComponent implements OnInit {
+	@ViewChild('inputRef', { static: true })
+	inputRef?: ElementRef<HTMLInputElement>;
```

Comment:
suggestion: since we say it's `static`, we can use Non-null Assertion operator instead